packages/world/mud.config.ts

```diff
@@ -227,13 +229,14 @@ export default defineWorld({
       },
       key: ["x", "y", "z"],
     },
-    MoveUnits: {
```

Comment:
since the table isnt deleted on upgrade, we should keep it but just move it below and add a deprecated comment

Reply:
will add back